application/src/main/java/org/togetherjava/tjbot/db/package-info.java

```diff
@@ -1,5 +0,0 @@
-/**
- * This package contains the database management of the application. See
- * {@link org.togetherjava.tjbot.db.Database} to get started.
- */
-package org.togetherjava.tjbot.db;
```

Comment:
Why was this file removed?

Reply:
I don't know, something went wrong during rebase, I'll try to fix

Reply:
That file doesn't even exist, the issue was with dirty history, it looks good now. ^^

Reply:
Also, SonarCloud is expected to fail, since this PR comes from a fork.